contributing_to_docs/templates/overview_topic_template.adoc

```diff
@@ -1,8 +1,31 @@
 = Overview
 {product-author}
 {product-version}
-:data-uri:
-:icons:
+:icons: font
 :experimental:
 
---Remove this and add a brief introduction here--
+--Start with a clear introduction to the topic and what information customers can expect to find in the topic--
```

Comment:
Also, it's probably not too important, but I feel it would work better with a slight combination of the two sentences, and for it to look like the next bit. As it is, it might not be clear enough for newbies to remove the "--" at the start, etc. Originally, I thought this was another way to put titles into topics (instead of =='s) 

So:

////
Remove this paragraph and add a clear introduction to the section and what information customers can expect to find here. 
////

With Alex's suggestion of changing "topic" to "section", but that may not be the best word. Chapter? Book?